azext_edge/edge/providers/support/base.py

```diff
@@ -512,3 +431,39 @@ def _capture_pod_container_logs(
                 logger.debug(e.body)
 
     return processed
+
+
+def _process_kubernetes_resources(
+    resources: Any,
```

Comment:
There is not a more specific base type ?

Reply:
Alternatively, Union would work adding lots of types (base type preferred)

Reply:
the base type used in their sdk is object, so I will change to that instead